src/components/FilterDrawer/FilterDrawer.js

```diff
@@ -51,10 +69,20 @@ const FilterDrawer = ({ children }) => {
 };
 
 FilterDrawer.propTypes = {
+  resourceTypeFilters: shape({}).isRequired,
+  toggleResourceTypeFilterAction: func.isRequired,
   children: node.isRequired,
 };
 
-export default FilterDrawer;
+const mapStateToProps = (state) => ({
+  resourceTypeFilters: supportedResourceTypeFiltersSelector(state),
+});
+
+const mapDispatchToProps = {
+  toggleResourceTypeFilterAction: toggleResourceTypeFilter,
```

Comment:
we should probably settle on a convention on `toggleResourceTypeFilter` vs `toggleResourceTypeFilterAction`, for both the key and value.

...I'd say `toggleResourceTypeFilter` for both, b/c when you see `mapDispatchToProps`, b/c it is implicitly an "action", based on `mapDispatchToProps` and other clues.